### ISSUE TITLE
Investigate turning this cookbook into a hybrid-library cookbook

### ISSUE THREAD

**Issue by tas50** [chef-cookbooks/kickstart#1]
The cookbook setup is very opinionated and requires forking to edit the config

**Comment by nkadel-skyhook** [comment]
Agreed. "/srv" is not an ideal location for the kickstart files, and the contents of the kickstart file template  are incompatible with RHEL 6 or RHEL 7.

**Comment by nandanhegde73** [comment]
This issue or pull request is being automatically closed because the repository is being archived as part of the [Repository Standardization Initiative](https://github.com/chef-boneyard/oss-repo-standardization-2025).
